File: scalaz/src/main/scala/org/openapitools/client/api/BadEventQueueIdError.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime


import BadEventQueueIdError._

case class BadEventQueueIdError (
  result: AnyType,
msg: AnyType,
code: Option[AnyType],
/* The string that identifies the invalid event queue.  */
  queueId: Option[String])

object BadEventQueueIdError {
  import DateTimeCodecs._

  implicit val BadEventQueueIdErrorCodecJson: CodecJson[BadEventQueueIdError] = CodecJson.derive[BadEventQueueIdError]
  implicit val BadEventQueueIdErrorDecoder: EntityDecoder[BadEventQueueIdError] = jsonOf[BadEventQueueIdError]
  implicit val BadEventQueueIdErrorEncoder: EntityEncoder[BadEventQueueIdError] = jsonEncoderOf[BadEventQueueIdError]
}

File: javascript-closure-angular/API/Client/BadEventQueueIdError.js
goog.provide('API.Client.BadEventQueueIdError');

/**
 * @record
 */
API.Client.BadEventQueueIdError = function() {}

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.BadEventQueueIdError.prototype.result;

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.BadEventQueueIdError.prototype.msg;

/**
 * @type {!API.Client.AnyType}
 * @export
 */
API.Client.BadEventQueueIdError.prototype.code;

/**
 * The string that identifies the invalid event queue. 
 * @type {!string}
 * @export
 */
API.Client.BadEventQueueIdError.prototype.queueId;

